### ISSUE TITLE
error 600 in open nc subset on some nc files

### ISSUE THREAD

**Issue by shantosa** [NetCDF4Excel/project#16]
I am using necdf4excel version 3.2, but I have difficulties on some file of nc. I can open the other nc file from cordex east asia. It works and fine. but I could not open some file from ISI-MIP project and has error message : "error 600 in open nc subset" . I have sent email to : netcdf4xl@gmail.com  a week ago, but there is no repply for my mail. I  attached the file on that mail to be checked. kindly please guide on this matter since I need the data for my dissertation.​

**Comment by jdrenken** [comment]
I have the same issue. Hopefully someone gets back to us!

**Comment by swordsgnat** [comment]
Yeah, same issue, commented with an example file in the newer thread

**Comment by fvallejog** [comment]
I think that this problem is because of the NetCDF version. I work with WRF output (air quality) and the add-in read very well a file generated with WRF3.9, but with the file created in WRF 4.1.5 (newer NetCDF) didn't work.

**Comment by fvallejog** [comment]
Change to NETCDF-3. It's the solution!!

**Comment by redhaven75** [comment]
> 
> 
> Change to NETCDF-3. It's the solution!!

thanks fvallejog, how to convert to netcdf-3? thanks
